etl/meta/collections/10027.testing-tool.yml

```diff
@@ -21,3 +21,4 @@ items:
   - microsoft/playwright
   - webdriverio/webdriverio
   - selenide/selenide
+  - mcollina/autocannon
```

Comment:
It seems that this tool is used for benchmarking, not for testing. Perhaps we could add a benchmark collection.